lib/ansible/modules/windows/win_nssm.ps1

```diff
@@ -96,9 +96,10 @@ Function Nssm-Remove
 
     if (Service-Exists -name $name)
     {
-        $cmd = "stop ""$name"""
-        $results = Nssm-Invoke $cmd
-
+        if ((get-service $name).Status -ne "Stopped") {
```

Comment:
I don't think it is, usually powershell is smart enough with parameters and knows what is in that variable is one value. I would change to `Get-Service -Name $name` though so we are explicitly setting the parameter to use. As always testing this would probably be best.